### PR TITLE
[DOC]Add kba reduce current consumption

### PR DESCRIPTION
**Pull request by silabs-JessicaZ**
**Description about the pull request**
- Add kba reduce current consumption 

**Review**
@MarkDing  (mentions of the person or team responsible for reviewing proposed changes. for e.g, @MarkDing)
(Note: Please make sure that all of the new source code in the pull request following the coding standard)

### CODE REVIEW

**Comment by silabs-JessicaZ** [comment]
Hi @suhuide, I add current consumption measurement kba to IoT-Developer-Boot-Camp-Wiki/Common repo. Could you please help review? Thanks in advance.

File: common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+The DC-DC buck converter covers a wide range of load currents, provides high efficiency in energy modes EM0, EM1, EM2 and EM3.
+```c
+  // Enable DC-DC converter
+  EMU_DCDCInit_TypeDef dcdcInit = EMU_DCDCINIT_DEFAULT;
+  EMU_DCDCInit(&dcdcInit);
+```
+ 
+##### GPIO
+All unconnected pins on the EFR32 should be configured to Disabled mode (high impedance, no pull resistor), and the reset state of the IO pins is disabled as well. This would be done by calling MX_25deinit().
+```c
```
Comment (silabs-EricS) [inline common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22.md:78]: This should be done by set GPIO gpioModeDisabled, not by call MX_25deinit(), this function only set SPI relevant pin.

File: common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22-CN.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+DC-DC降压变换器涵盖广泛的负载电流范围，在EM0, EM1, EM2和EM3能量模式下提供高效率。
+```c
+  // Enable DC-DC converter
+  EMU_DCDCInit_TypeDef dcdcInit = EMU_DCDCINIT_DEFAULT;
+  EMU_DCDCInit(&dcdcInit);
+```
+ 
+##### GPIO
+所有未连接的引脚在EFR32上应配置到 Disable模式(高阻抗，无外接电阻)，和reset相关的IO引脚也是禁用的。这可以通过调用MX_25deinit()来完成。
+```c
```
Comment (silabs-EricS) [inline common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22-CN.md:78]: 与E文版相同，要用gpioModeDisabled而非MX_25deinit()，可以说参照MX_25deinit()。

File: common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22-CN.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+ 
+
+# 测试实例
+以下实例项目采用了上述大部分策略来降低能耗。
+ 
+##### 实验示例
+附加的示例项目可用于重现本文中所示的结果。
+ 
+1.使用版本2.13.6或更新版本的Bluetooth SDK创建一个新的SoC - Empty应用程序项目。
+2.打开app.c并注释掉system_boot中的代码禁止广播，以便我们可以测量EM2中的睡眠电流。
```
Comment (silabs-EricS) [inline common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22-CN.md:127]: 1./2. 在同一行了

File: common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+ 
+The figure below shows the result tested from Energy Profiler in Simplicity Studio. From the view, we can find two current measuring results. The left one represents the total average current which include the huge increasing of current while reset. And the right one calculated from the selected range should be the values user read from.
+ 
+![common](files/CM-Reduce-Current-Consumption/SoC_Empty_EM2current_consumption.png)  
+ 
+Enable or disable different peripherals will have an impact on the current consumption. To reduce current draw, it is recommended to use adjusted   voltage to  optimize the energy efficiency of the system. We will discuss later the effect of different peripherals on current consumption.
+ 
+##### VCOM 
+If you are reproducing the EM2 current consumption test using some of the example come with our SDK (either MCU or Wireless), please also check the status of the VCOM. Enable VCOM will increase the current consumption. Deassert the EN pin as well as the TX and RX pins when not needed.
+```c
```
Comment (silabs-EricS) [inline common/Measure-EM2-Current-Consumption-under-1.4uA-in-EFR32xG22.md:21]: EN pin, here need a clarify.

**Comment by MarkDing** [comment]
Duplicate pull request

**Pull request closed without merging**